fmriprep/interfaces/tests/test_reports.py

```diff
@@ -0,0 +1,35 @@
+import nibabel as nb
+import nibabel.orientations as ornt
+import numpy as np
+import pytest
+
+from ..reports import get_world_pedir
+
+
+def generate_img(orientation):
+    dataobj = np.zeros((2, 2, 2), dtype='<i1')
+    affine = np.eye(4)
+    rot = ornt.inv_ornt_aff(ornt.axcodes2ornt(orientation), (1, 1, 1))
+    affine[:3, :3] = np.diag((1, 1, 1)) @ np.linalg.inv(rot[:3, :3])
+    return nb.Nifti1Image(dataobj, affine)
+
+
+@pytest.mark.parametrize("orientation,pe_dir,expected", [
+    ('RAS', 'j', 'Posterior-Anterior'),
+    ('RAS', 'j-', 'Anterior-Posterior'),
+    ('RAS', 'i', 'Left-Right'),
+    ('RAS', 'i-', 'Right-Left'),
+    ('RAS', 'k', 'Inferior-Superior'),
+    ('RAS', 'k-', 'Superior-Inferior'),
+    ('LAS', 'j', 'Posterior-Anterior'),
+    ('LAS', 'i-', 'Left-Right'),
+    ('LAS', 'k-', 'Superior-Inferior'),
+    ('LPI', 'j', 'Anterior-Posterior'),
+    ('LPI', 'i-', 'Left-Right'),
+    ('LPI', 'k-', 'Inferior-Superior'),
+])
+def test_get_world_pedir(tmpdir, orientation, pe_dir, expected):
+    img = generate_img(orientation)
+    assert orientation == ''.join(nb.aff2axcodes(img.affine))
```

Comment:
I guess we don't really need `generate_img` if we're using the orientation and not the image in `get_world_pedir`.